qlib/utils/resam.py

```diff
@@ -273,8 +329,9 @@ def resam_ts_data(
         end sampling time, by default None
     method : Union[str, Callable], optional
         sample method, apply method function to each stock series data, by default "last"
-        - If type(method) is str, it should be an attribute of SeriesGroupBy or DataFrameGroupby, and run feature.groupby
-        - If `feature` has MultiIndex[instrument, datetime], method must be a member of pandas.groupby when it's type is str.or callable function.
+        - If type(method) is str or callable function, it should be an attribute of SeriesGroupBy or DataFrameGroupby, and applies groupy.method for the sliced time-series data
+        - If method is None, do nothing for the sliced time-series data.
+        - Only when the index `feature` is MultiIndex[instrument, datetime], the method is valid.
```

Comment:
You didn't give a accurate description for  return value `None`

Reply:
`Only when the index `feature` is MultiIndex[instrument, datetime], the method is valid.`
I can't understand this line of the document.

Reply:
Maybe
Check XD's PR